src/amo/components/CollectionList/index.js

```diff
@@ -0,0 +1,71 @@
+/* @flow */
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+import CollectionManager from 'amo/components/CollectionManager';
+import { getCurrentUser } from 'amo/reducers/users';
+import AuthenticateButton from 'core/components/AuthenticateButton';
+import translate from 'core/i18n/translate';
+import Card from 'ui/components/Card';
+import type { UsersStateType } from 'amo/reducers/users';
+import type { I18nType } from 'core/types/i18n';
+import type { ReactRouterLocation } from 'core/types/router';
+
+import './styles.scss';
+
+
+export type Props = {|
+  creating: boolean,
+  i18n: I18nType,
+  isLoggedIn: boolean,
+  location: ReactRouterLocation,
+|};
+
+export class CollectionListBase extends React.Component<Props> {
+  static defaultProps = {
+    creating: false,
+  };
+
+  renderManager() {
+    const { creating, i18n, isLoggedIn, location } = this.props;
+    if (creating) {
+      if (!isLoggedIn) {
+        return (
+          <AuthenticateButton
+            noIcon
+            location={location}
+            logInText={i18n.gettext('Sign in to create a collection')}
```

Comment:
Do we use "Sign in" anywhere else on the site? I think this should be "Log in", for consistency.

Reply:
It's used in [`Collection`](https://github.com/mozilla/addons-frontend/blob/ceb360b6ab5358eee52baebca0f7d49202409f01/src/amo/components/Collection/index.js#L175), which is where I got the idea, but you're correct that elsewhere on the site we use "Log in".  I'll change it here and for `Collection` as well.